docs/Agent/config/config.md

```diff
@@ -0,0 +1,22 @@
+### Configuration
+
+**Note**: This Documentation is only required for administrative purposes.
+By Default the collected results are moved to pbench-server.
+
+
+#### Configure with Server < 1.0
+
+For the purpose of defining the required configuration, a pbench-agent configuration file must be created. The config file and the ssh key file must be present in the specified place for the ansible roles to function.
+
+The installation includes a sample configuration file at '/opt/pbench-agent/config/pbench-agent.cfg'. Make a backup of this file, then update the lines marked with # CHANGE ME! comments to suit your setup. Please make sure to make this file accessible to users.
+
+The ssh key pair can be generated with:
+
+	ssh-keygen -t rsa
+
+with an empty passphrase. The private key must be made available to users before they can complete the installation of pbench-agent as stated above. The authorized_keys list should include the public key.
```

Comment:
I don't find this particularly useful. First off, if you're giving the specific command to generate a key pair, shouldn't we be saying what to *do* with them? The public key needs to be in the `pbench` account `~/.ssh/authorized_keys` on the Pbench Server, while the private key needs to be in `/opt/pbench-agent/id-rsa` on each Pbench Agent installation.

There's also the point that this section loosely implies actions on both the Pbench Agent *and* the Pbench Server, but the reference to "The installation" isn't clear.

I'd also hope we can drop all mention of 0.69 Pbench Server from `main`, but sadly that would probably be premature as we'll likely continue to support that for at least 0.73.